### DISCUSSION TITLE
How to centering `next/image`'s <Image>?

### DISCUSSION THREAD

**Discussion by ogawa0071** [vercel/next.js#18375] [Help] [answered]
```
import Image from "next/image";

<Image
src="/image.png"
width="1000"
height="1000"
style={{
  margin: "auto",
}}
></Image>
```

This code is not work to centering in borwser window width > 1000px.

**Comment by TimmNL** [comment, marked as answer]
The way I got around this is by adding a wrapper div. Reason being that the styling you added is placed on the `img` tag instead of the wrapper;
```
<div style="max-width:100%;width:1000px">
    <div style="position:relative;padding-bottom:100%">
        <img
            src="/_next/image?url=/image.png&amp;w=420&amp;q=75"
            srcset="/_next/image?url=/image.png&amp;w=320&amp;q=75 320w, /_next/image?url=/image.png&amp;w=420&amp;q=75 420w"
            style="visibility:visible;height:100%;left:0;position:absolute;top:0;width:100%"  // <-- your style is added here
        />
    </div>
</div>
```

By wrapping the Image component with a container you can position it in the center again.
If there is a better way, I'd like to hear it as well, but this worked for me :)

**Comment by ogawa0071** [comment]
@TimmNL 
Thanks!

I rewrite code, and it works!

```typescript
import Image from "next/image";

export default function ImageSample() {
  return (
    <>
      <div
        style={{
          display: "flex",
          justifyContent: "center",
        }}
      >
        <Image src="/image.png" width="100" height="1000" />
      </div>
    </>
  );
}
```

![image](https://user-images.githubusercontent.com/3258736/97445385-8c8cc780-1970-11eb-9800-1f87fed05ace.png)

**Reply by bobthebuilder1997** [reply]
Thanks. This helped me.

**Reply by saydur961** [reply]
Thank you so much

**Reply by aralroca** [reply]
And is not possible without add an extra HTML node?

**Reply by bobarros** [reply]
You can write some js to edit the original one, but it's going to be messy.

**Reply by aralroca** [reply]
So instead of 1 HTML node: `<img style={{ display: 'block', margin: '0 auto'}} />` you need 1 extra div + 2 divs that the Image component is already adding, finally 4 HTML nodes for each image...

I don't know if the Image component really helps performance, maybe it's better to write the `srcSet` manually.

**Reply by TheoOliveira** [reply]
in my opinion a cleaner way <img width="649" alt="Uploaded using RayThis Extension" src="https://user-images.githubusercontent.com/11945175/134672892-7af35078-5db2-4cf3-9b02-9f8000209298.png">

**Reply by dogomayana** [reply]
thanks man for this

**Comment by eldoy** [comment]
Vanilla HTML: `<img src="/img/logo.png">` in a `text-align: center` container, no NextJS needed, in fact, no libs at all. Please, for the sake of humanity, stop using React and Next.
